web-common/src/components/column-profile/column-types/sparks/TimestampSpark.svelte

```diff
@@ -14,59 +14,17 @@
     Line,
   } from "@rilldata/web-common/components/data-graphic/marks";
 
-  //export let data;
-
   export let width = undefined;
   export let height = undefined;
   export let data: unknown[];
-  // export let curve = "curveLinear";
-  // export let area = false;
-  // export let color = "hsl(217, 10%, 50%)";
-  // export let tweenIn = false;
-
-  // // the color of the zoom window
-  // export let zoomWindowColor = "hsla(217, 90%, 60%, .2)";
-  // // the color of the zoom window boundaries
-  // export let zoomWindowBoundaryColor = "rgb(100,100,100)";
-  // export let zoomWindowXMin: Date = undefined;
-  // export let zoomWindowXMax: Date = undefined;
```

Comment:
Note: changes in this file are just opportunistic cleanup that I found while searching "color"  this code was commented out 12-14 months ago.